lib/generators/react_on_rails/base_generator.rb

```diff
@@ -165,7 +165,7 @@ def append_to_assets_initializer
           Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
           Rails.application.config.assets.paths << Rails.root.join("client", "assets", "images")
           Rails.application.config.assets.paths << Rails.root.join("client", "assets", "fonts")
-          Rails.application.config.assets.precompile += %w( generated/server-bundle.js )
+          Rails.application.config.assets.precompile += %w( generated/server-bundle.js react_on_rails )
```

Comment:
There's many other gems that provide JS files (take jquery-rails, for example).

Do they also put them in the precompile list?